riptable/rt_fastarray.py

```diff
@@ -1246,6 +1246,28 @@ def _reduce_check(self, reduceFunc: REDUCE_FUNCTIONS, npFunc, *args, **kwargs):
             result = npFunc(self._np, *args, **kwargs)
             return result
 
+        if reduceFunc in [
+                REDUCE_FUNCTIONS.REDUCE_MIN,
+                REDUCE_FUNCTIONS.REDUCE_NANMIN,
+                REDUCE_FUNCTIONS.REDUCE_MAX,
+                REDUCE_FUNCTIONS.REDUCE_NANMAX,
+                REDUCE_FUNCTIONS.REDUCE_MEAN,
+                REDUCE_FUNCTIONS.REDUCE_NANMEAN ]:
+            if  self.dtype not in [

```

Comment:
If you look at the dtype heirarchy (https://numpy.org/doc/stable/reference/arrays.scalars.html) this may be simplified to
```
if not np.issubdtype(self.dtype, np.numeric): ...
```